include/seqan3/alignment/pairwise/alignment_result.hpp

```diff
@@ -130,17 +146,20 @@ class alignment_result
      * \{
      */
 
-    /*!\brief Constructs a seqan3::alignment_result from an `alignment_result_traits` object.
+    //!\privatesection
```

Comment:
No, this is intended. The better design would be to have a factory class that is a friend of this class and can invoke this constructor which should be in fact private. This comes later. For now it is ok to just document it private so that users won't see this in the API docs.